src/DRICH_geo.cpp

```diff
@@ -455,7 +482,7 @@ static Ref_t createDetector(Detector& desc, xml::Handle_t handle, SensitiveDetec
     auto sensorSphPos = Position(sensorSphCenterX, 0., sensorSphCenterZ) + originFront;
 
     // sensitivity
-    if (!debugOptics || debugOpticsMode == 3)
+    if (!debugOptics || debugOpticsMode == 3 || debugOpticsMode == 4)
```

Comment:
Avoid relying implicitly on the boolean conversion of integers. Suggestion: use `debugOptics != 0`.

Reply:
This applies in multiple places.